dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/ShadowClassLoader.java

```diff
@@ -126,6 +126,10 @@ private static ClassLoader getShadow(ClassLoader loader) {
     protected Class<?> findClass(final String name) throws ClassNotFoundException {
         final ByteResourceInformation classBytesResourceInformation = shadowedLoader.findClassBytes(name);
 
+        if (classBytesResourceInformation == null) {
+            throw new ClassNotFoundException(name);
```

Comment:
Here we are missing some of the useful message information from the exception that got removed.

Reply:
The only thing missing from the exception is the resource name.  I could put it back into this path, but it will have limited value in my opinion which is why didn't put it in.  The class name is the important thing to be in the exception and it is there.

If I were to put it back in it would look like this:
```
    if (classBytesResourceInformation == null) {
        String resourceName = Util.convertClassNameToResourceName(className);
        String message = String.format("Could not find class '%s' as resource '%s'", name, resourceName);
        throw new ClassNotFoundException(message);
    }```